include/mbgl/style/source.hpp

```diff
@@ -75,6 +75,15 @@ class Source : public mbgl::util::noncopyable {
     virtual void loadDescription(FileSource&) = 0;
     void setPrefetchZoomDelta(optional<uint8_t> delta) noexcept;
     optional<uint8_t> getPrefetchZoomDelta() const noexcept;
+
+    // Sets a limit for how much parent tile can be overscaled.
+    //
+    // When a set of tiles for a current zoom level is being rendered and some of the
+    // ideal tiles that cover the screen are not yet loaded, parent tile could be
+    // used instead. This might introduce unwanted rendering side-effects, especially
+    // for raster tiles that are overscaled multiple times.
```

Comment:
This explanation of why a developer may want to set a `maxOverscaleFactor` for a raster source is good. 

My suggestions are:
- Add information about what the default behavior is if nothing is set
- Is there any further guidance we could provide to help developers determine which number they may want to consider setting as the factor? Commented a possible example below (`For example...`)
- Could we provide more guidance on when/how/which parent tiles might be available for this purpose?

Reply:
```suggestion
    // When a set of tiles for the current zoom level is being rendered and one or more
    // ideal tiles that cover the screen have not yet loaded, parent tiles available
    // in XYZ cache could be used instead. This might introduce unwanted rendering 
    // side effects, especially if raster tiles are overscaled multiple times.
    
    // For example, a `maxOverscaleFactor` of 3 would mean that on zoom level 13, the
    // minimum zoom level of a parent tile that could be used in place of an ideal
    // tile during rendering would be zoom XX. By default, no limit is set, so any
    // parent tile may be used.
```

Reply:
@chloekraw thank you! I added example and omitted notion of cache, it would complicate description.